scripts/qiita

```diff
@@ -289,6 +294,49 @@ def update_preprocessed_data(sl_out_dir, study,  preprocessed_data):
                                             preprocessed_data)
     click.echo("Preprocessed data %s successfully updated" % ppd.id)
 
+# #############################################################################
+# PORTAL COMMANDS
+# #############################################################################
+
+
+@portal.command(name="add-analyses")
+@click.option('--portal', required=True, type=str,
+              help='Portal to add the analyses to')
+@click.option('--analyses', required=True, type=str,
```

Comment:
Doable. and with the portal editing already implied in the command it makes sense.

> On Jul 1, 2015, at 9:18 AM, Yoshiki Vázquez Baeza <notifications@github.com> wrote:
> 
> In scripts/qiita <https://github.com/biocore/qiita/pull/1316#discussion_r33695442>:
> 
> > @@ -289,6 +294,49 @@ def update_preprocessed_data(sl_out_dir, study,  preprocessed_data):
> >                                              preprocessed_data)
> >      click.echo("Preprocessed data %s successfully updated" % ppd.id)
> >  
> > +# #############################################################################
> > +# PORTAL COMMANDS
> > +# #############################################################################
> > +
> > +
> > +@portal.command(name="add-analyses")
> > +@click.option('--portal', required=True, type=str,
> > +              help='Portal to add the analyses to')
> > +@click.option('--analyses', required=True, type=str,
> Got it, did not know that. Maybe both options should become arguments? ...
> … <x-msg://75/#>
> —
> Reply to this email directly or view it on GitHub.
> 
>  <https://github.com/biocore/qiita/pull/1316/files#r33694871>
> —
> Reply to this email directly or view it on GitHub <https://github.com/biocore/qiita/pull/1316/files#r33695442>.
>